src/docs/sdk/event-payloads/properties/measurements.mdx

```diff
@@ -27,8 +27,7 @@ Standard measurement keys currently supported are from the following list taken
 ]
 ```
 
-Custom measurements need units to be specified. Units currently supported are durations from the list `["ns" , "ms" , "s"]` taken from [here](https://github.com/getsentry/relay/blob/1e45a8817e45408ecbdbd3f0a679a5f5e4885290/relay-metrics/src/protocol.rs#L14-L21). Note that in the future we will extend the available units (for instance, memory units) but we will not implement a generic user-defined unit interface.
-
+Custom measurements need units to be specified. The full list of supported units is specified on [Relay's `MetricUnit`](https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html). Sentry's event ingestion supports arbitrary custom units, but many SDKs will not expose a generic user-defined unit interface.
```

Comment:
We'll have to change the `measurements` json example below to use the full name for `ms`

Reply:
Thanks, updated!